src/reanimated2/animations.js

```diff
@@ -174,7 +174,7 @@ export function withSpring(toValue, userConfig, callback) {
 
       const isOvershooting = () => {
         if (config.overshootClamping && config.stiffness !== 0) {
-          return current < toValue ? current > toValue : current < toValue;
+          return current < toValue ? animation.current > toValue : animation.current < toValue;
```

Comment:
```suggestion
          return current < toValue
            ? animation.current > toValue
            : animation.current < toValue;
```

Could you format it accordingly?

Reply:
done, thks :)